wiki/Ranking_criteria/Simplified_ranking_criteria/en.md

```diff
@@ -91,6 +91,7 @@ The **simplified ranking criteria** aims to give mappers a clearer perspective o
 
 ### Song
 
+- **Make sure the song adheres to [content usage permissions](/wiki/Rules/Content_usage_permissions#artist-permissions)**
```

Comment:
huh this reads weird, because you'd expect "guidelines" or "rules" and not "permissions"

or maybe the article should be renamed back to "content usage guidelines"?

Reply:
welp i just used the article name verbatim because that made the most sense

Reply:
```suggestion
- **All beatmap content must be cleared regarding [content usage permissions](/wiki/Rules/Content_usage_permissions#artist-permissions).**
```

and move to the "general" section